### DISCUSSION TITLE
Run the script on fresh windows install or after everything ist setup?

### DISCUSSION THREAD

**Discussion by f00tf00t** [simeononsecurity/Windows-Optimize-Harden-Debloat#64] [Q&A] [answered]
As per the title; is it best to run the script on a fresh windows install (before any customizations and/or apps are installed) or after?

Thank you

**Comment by simeononsecurity** [comment, marked as answer]
> As per the title; is it best to run the script on a fresh windows install (before any customizations and/or apps are installed) or after?
> 
> Thank you

You should always review and test the script to the best of your abilities before running it on any system you plan to use.
This goes for any script online, not just ours.

The script works best on a clean system. However, there is nothing wrong with running it otherwise assuming you've tested it.

Please be sure to read the notes and requirements before running it @f00tf00t
